test/init/projectAutoDetection.test.ts

```diff
@@ -67,7 +67,9 @@ describe('AutoDetectProject: tests', function () {
         function () {
           userConfig.projectDetection.disableDetect = true;
           return pad.detectProject(null, userConfig)
-            .then(res => expect(res).to.be.an('array'));
+            .then(res => {
```

Comment:
@robertohuertasm Is this your prefered coding style?

Reply:
No. I was using this to debug and later I forgot to go back to a one line style. It was late... 😉